docs/core/containers/publish-configuration.md

```diff
@@ -427,10 +430,23 @@ Where possible, existing MSBuild properties provide the values for these labels.
 | `org.opencontainers.image.title`                                                     |                                                                                                    | `ContainerTitle`             | `Title`                    | `ContainerGenerateLabelsImageTitle`         |                                                                                                                     |
 | `org.opencontainers.image.base.name`                                                 |                                                                                                    | `ContainerBaseImage`         |                            | `ContainerGenerateLabelsImageBaseName`      |                                                                                                                     |
-| `org.opencontainers.image.base.digest`                                               |                                                                                                    |                              |                            | `ContainerGenerateLabelsImageBaseDigest`    | This will be the SHA digest of the chosen base image. Available from .NET SDK 9.0.100 onwards.                      |
+| `org.opencontainers.image.base.digest`                                               |                                                                                                    |                              |                            | `ContainerGenerateLabelsImageBaseDigest`    | This is the SHA digest of the chosen base image. Available from .NET SDK 9.0.100 onwards.                      |
 | `org.opencontainers.image.source`                                                    |                                                                                                    | `PrivateRepositoryUrl`       |                            | `ContainerGenerateLabelsImageSource`        | Only written if `PublishRepositoryUrl` is `true`. Also relies on Sourcelink infrastructure being part of the build. |
 | `org.opencontainers.image.revision`                                                  |                                                                                                    | `SourceRevisionId`           |                            | `ContainerGenerateLabelsImageRevision`      | Only written if `PublishRepositoryUrl` is `true`. Also relies on Sourcelink infrastructure being part of the build. |
 
+## Multi-architecture images
+
+Multi-architecture images enable a single container image to support multiple architectures, simplifying cross-platform development and deployment. The .NET SDK supports this through the `ContainerRuntimeIdentifiers` property.
```

Comment:
So there's some subtlety here with ContainerRuntimeIdentifiers vs RuntimeIdentifiers that I want to touch on. 

`ContainerRuntimeIdentifiers` _must_ be a sub-set of `RuntimeIdentifiers` or else core parts of then pipeline will break. We added `ContainerRuntimeIdentifiers` to cover scenarios when a user wants to target many RIDs _in general_ (e.g. making an AOT Windows version of their app), but some of those RIDs don't support containerization (like macos RIDs).

All of the rest of this section is great - I just didn't know when/where to talk about the intended use of ContainerRuntimeIdentifiers vs RuntimeIdentifiers.

Reply:
@baronfel I added to the note on line 191. How's that?

Reply:
Actually I added another note in the *Multi-architecture images* section, too.